smart-py/objkt_swap_v3_test.py

```diff
@@ -885,96 +718,14 @@ def test_swap_failure_conditions():
             operator=marketplaceV3.address,
             token_id=152))]).run(sender=artist1)
 
-    # Try to Swap the OBJKT as someone else with a different creator must fail
-    scenario += marketplaceV3.swap(
-        fa2=objkt.address,
-        objkt_id=152,
-        objkt_amount=1,
-        xtz_per_objkt=sp.mutez(10000),
-        royalties=100,
-        creator=artist2.address
-    ).run(
-        valid=False,
-        sender=artist2
-    )
-
-    # No swap was added
-    scenario.verify(marketplaceV3.data.swaps.contains(0) == False)
-    scenario.verify(marketplaceV3.data.counter == 0)
-
-    # Try to Swap the OBJKT as someone else with the same creator must fail
-    scenario += marketplaceV3.swap(
-        fa2=objkt.address,
-        objkt_id=152,
-        objkt_amount=1,
-        xtz_per_objkt=sp.mutez(10000),
-        royalties=100,
-        creator=artist1.address
-    ).run(
-        valid=False,
-        sender=artist2
-    )
```

Comment:
is this a dupe?

Reply:
Here is also that it fails because artist2 didn't give the operators right. Of course it would fail later becase it doesn't have the token. But that's more a FA2 test, rather than a test of v3